tests/cpu/CMakeLists.txt

```
@@ -114,6 +114,7 @@ if(OCIO_ADD_EXTRA_BUILTINS)
     set(SOURCES_BUILTINS
         transforms/builtins/ArriCameras.cpp
         transforms/builtins/CanonCameras.cpp
+		transforms/builtins/Displays.cpp
```

```suggestion
	transforms/builtins/Displays.cpp
```
"Sorry, but i'm a bit of a stickler for paperwork..."